### PR TITLE
Cluster lifecycle new topic

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/placement_managed.adoc [hunk 1]
```diff
@@ -4,21 +4,22 @@
 A `Placement` resource is a namespace-scoped resource that defines a rule to select a set of `ManagedClusters` from the `ManagedClusterSets`, which are bound to the placement namespace.
 
 **Required access**: Administrator
+//check this
```
Comment (swopebe) [inline clusters/placement_managed.adoc:7]: I need to know the RBAC info

Comment (elgnay) [reply clusters/placement_managed.adoc:7]: ClusterSet Admin is fine instead of Administrator.

File: clusters/placement_managed.adoc [hunk 2]
```diff
@@ -4,21 +4,22 @@
...
 - Kubernetes clusters are registered with the hub cluster as cluster-scoped `ManagedClusters`.
 
-- `ManagedClusters` are organized into cluster-scoped `ManagedClusterSets`.
+- The `ManagedClusters` are organized into cluster-scoped `ManagedClusterSets`.
 
-- `ManagedClusterSets` are bound to workload namespaces.
+- The `ManagedClusterSets` are bound to workload namespaces.
 
-- Namespace-scoped `Placements` specify a portion of `ManagedClusterSets` which select a working set of the potential `ManagedClusters`.
+- The Namespace-scoped `Placement` specifies a portion of `ManagedClusterSets` that select a working set of the potential `ManagedClusters`.
```
Comment (elgnay) [inline clusters/placement_managed.adoc]: Namespace-scoped -> namespace-scoped

File: clusters/placement_managed.adoc [hunk 3]
```diff
@@ -144,7 +142,8 @@ spec:
 - If a `ManagedCluster` is selected and added into a - `PlacementDecision` components , which consume this `Placement`, may apply workload on this `ManagedCluster`; once the `ManagedCluster` is no longer selected and removed from the `PlacementDecisions`, the workload applied on this `ManagedCluster` should be evicted accordingly.
```
Comment (elgnay) [inline clusters/placement_managed.adoc]: If a `ManagedCluster` is selected and added into a `PlacementDecision`, component consuming this `Placement` may apply workload on this `ManagedCluster`; once the `ManagedCluster` is no longer selected and removed from the `PlacementDecisions`, the workload applied on this `ManagedCluster` should be evicted accordingly.

File: clusters/placement_managed.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/placement_managed.adoc:7]: thanks for that

**Pull request merged**